README.md

```diff
@@ -39,17 +39,19 @@ This project uses [SEMVER](https://semver.org/).
 |      2.0.1     |    X    |   X   |  X  |    X    |  X  |
 |      2.0.2     |    X    |   X   |  X  |    X    |  X  |
 |      3.x.x     |    X    |   X   |  X  |    X    |  X  |
+|      4.x.x     |    X    |   X   |  X  |         |     |
```

Comment:
I think we won't be able to support mobile _just yet_ (perhaps we can?)

Reply:
Why ?

Reply:
My bad! I confused it with HTML5 - I'll fix this.